app/services/unsplash_service.rb

```diff
@@ -0,0 +1,14 @@
+class UnsplashService < BaseService
+  def fetch_background(location)
+    path = '/search/photos/?Accept-Version=v1'
+    params = { client_id: ENV['UNSPLASH_API'], query: location}
```

Comment:
Added `UNSPLASH_API` to the application.yml. I realize all of the API keys should be renamed to `KEY `instead of `API `to reduce future developer confusion